website/docs/language/settings/backends/remote.html.md

```diff
@@ -8,12 +8,18 @@ description: |-
 
 # remote
 
-**Kind: Enhanced**
-
--> **Note:** We recommend using Terraform v0.11.13 or newer with this
-backend. This backend requires either a Terraform Cloud account on
-[app.terraform.io](https://app.terraform.io) or a Terraform Enterprise instance
-(version v201809-1 or newer).
+-> **Note:** With the release of Terraform v1.1.0, we recommend using the Terraform Cloud's built-in
+[`cloud` integration](/docs/language/settings/terraform-cloud/index.html) instead of this backend.
+The `cloud` option includes an improved user experience and more features.
+
+-> **Note:** This backend is unique among all other Terraform backends in that it has the ability to
+execute operations for Terraform Cloud's [CLI-driven run workflow](/docs/cloud/run/cli.html), rather
+than only store state snapshots. This has been previously referred to as being 'enhanced' backend
+behavior. Due to confusion about this specific distinction, this terminology has been retired.
```

Comment:
```suggestion
than only store state snapshots. (The documentation used to refer to this as "enhanced" backend
behavior, but it's simpler to describe it as a quirk of the `remote` backend.)
```

A three-high stack of "Note:" callouts is a bit tall; I'm fine with it for now, but we might also look to turn the "remote = special" one into a subsection below.